app/src/main/java/com/hedvig/app/feature/settings/Market.kt

```diff
@@ -61,9 +62,14 @@ enum class Market {
 
   fun openOnboarding(context: Context) {
     val webPath = Language.fromSettings(context, this).webPath()
-    val url = context.getString(R.string.WEB_BASE_URL) + "/" + webPath + "/new-member"
-    val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
-    context.startActivity(browserIntent)
+    val uri = Uri.parse("""${context.getString(R.string.WEB_BASE_URL)}/$webPath/new-member""")
+    val browserIntent = Intent(Intent.ACTION_VIEW, uri)
+    if (browserIntent.resolveActivity(context.packageManager) != null) {
+      context.startActivity(browserIntent)
+    } else {
+      e { "Tried to launch $uri but the phone has nothing to support such an intent." }
+      context.makeToast(hedvig.resources.R.string.NETWORK_ERROR_ALERT_MESSAGE)
```

Comment:
Can we use a more suitable message here?

Reply:
Yeah I could not find something in lokalise that would suit this here. If we want to create a new string then sure we could do that, and then wait for translations

Reply:
Gonna go with
general_unknown_error -> "Unknown Error"
as discussed 👍